### ISSUE TITLE
[Nav Redesign] Revise app copy as needed

### ISSUE THREAD

**Issue by jessex** [Recidiviz/pulse-dashboard#327]
**Is your feature request related to a problem? Please describe.**
We will roll out the navigation redesign feature iteratively, in a way that will not break functionality for the app between commits, in case a release needs to go out in the interim.

The mini-PRD only goes so far as reorganizing existing charts into new views that match the updated conceptual hierarchy--it does not yet propose new charts. This third task makes any copy updates necessitated by the project.

**Describe the solution you'd like**
1. Update important notes copy per PRD (to be supplied)
1. Update any chart titles and dynamic text as needed
1. Update any goal-oriented text as needed
1. Update any methodology text as needed

**Describe alternatives you've considered**
N/A

**Additional context**
This is a child of #324.

**Comment by jessex** [comment]
@kbinney, we'll need all of the final copy before this task happens.

**Comment by jessex** [comment]
@kbinney we need the final copy for this one so @DmitryJakubovsky can complete this soon. Thanks!

**Comment by kbinney** [comment]
Created a document for this accessible [here](https://docs.google.com/document/d/1SS0KnvpDzqclbvnWF11cphwpkKtyEDlvaPN2D7vQsck/edit#). This is organized by chart. Each chart and the Important Notes title has the "Methodology" text listed under it that I'd like used. The version in development didn't have any of the text for methodology or important notes re-used yet from what I could tell, so this doc is comprehensive for those components rather than a delta from the current version.  I also added notes where there are alerts, where any titles should be changed, and where any axis should be changed. 

The alerts are generally exactly the same as in the existing PRs. The titles are mostly the same, but the various "Revocations by X" charts are renamed "Revocation Admissions by X" (Listed chart by chart specifically). 

Finally, the axis labels have not changed except I noticed inconsistencies in the use of full or abbreviated month names on the x axis. They should all be changed to use abbreviated month names. This will help with keeping the charts more visible when the page size is shrunk.
